tests/unit/spec/base/search-params.js

```diff
@@ -42,6 +42,29 @@ describe('search-params.js', function() {
             expect(params.toString()).toEqual('scene=3&source=getfirefox');
         });
 
+        it('should return an object of utm_ values', function () {
+            var sp = new _SearchParams('utm_dude=lebowski&utm_sport=bowling&source=getfirefox');
+            var utms = sp.utmParams();
+            var keys = Object.keys(utms);
+            expect(keys).toEqual(['utm_dude', 'utm_sport']);
+            expect(utms.utm_dude).toEqual('lebowski');
+            expect(utms.utm_sport).toEqual('bowling');
+        });
+
+        it('should return an object of utm_ values with defaults for FxA', function () {
+            var sp = new _SearchParams('utm_dude=lebowski&utm_sport=bowling&source=getfirefox');
+            var utms = sp.utmParamsFxA('/es-ES/firefox/sync/');
```

Comment:
In non-testing use, no param will be sent to `utmParamsFxA`. The value will be set in the function to `window.location.pathname`, which doesn't contain any of the querystring. The testing value here is meant to mimic `pathname`, not a full URL.